.pre-commit-config.yaml

```diff
@@ -5,13 +5,13 @@ repos:
   - id: black
 
 - repo: https://github.com/nbQA-dev/nbQA
-  rev: 0.4.0
+  rev: 0.4.1
   hooks:
   - id: nbqa-black
     additional_dependencies: [black==20.8b1]
     args:
       - "--nbqa-mutate"
-    exclude: 00-intro.ipynb|05-pybind.ipynb
+    exclude: 05-pybind\.ipynb$
```

Comment:
this takes a regular expression, so `.` will match anything (rather than a literal dot)

Reply:
Ah, yes, you are right. 🤦